File: krscript/src/main/java/com/omarea/krscript/model/ActionInfo.java
package com.omarea.krscript.model;

import com.omarea.krscript.config.ActionParamInfo;

import java.util.ArrayList;

public class ActionInfo extends ConfigItemBase {
    public String descPollingShell;
    public String script;
    public String start;
    public ArrayList<ActionParamInfo> params;
}

File: krscript/src/main/java/com/omarea/krscript/model/ActionInfo.kt
package com.omarea.krscript.model

import com.omarea.krscript.config.ActionParamInfo
import java.util.*

class ActionInfo : ConfigItemBase() {
    var getState: String? = null
    var setState: String? = null
    var params: ArrayList<ActionParamInfo>? = null
}
